sui-execution/latest/sui-adapter/src/execution_engine.rs

```diff
@@ -996,6 +1001,29 @@ mod checked {
         builder
     }
 
+    fn setup_bridge_create(
+        mut builder: ProgrammableTransactionBuilder,
+    ) -> ProgrammableTransactionBuilder {
+        let bridge_uid = builder.programmable_move_call(
```

Comment:
any reason why we want to call into move to find a constant value?
I am not even clear who is using it in move.
thoughts?

Reply:
This is recommended by @emmazzz to get the custom object uid instead of making the function public(friend).

Reply:
how is this different from directly constructing an `ObjectArg`?

Reply:
right what @longbowlu said, that was the question, not sure there is a reason thus asking

Reply:
System txns allow for arbitrary function calls. I would just make the UID directly and skip the call

Reply:
I suggested this because this has been our programming pattern for creating system objects in a non-0x2 package: adding a private vanity ID creation function in `object`, calling the creation function with this id in the same PTB. Making the UID directly would certainly work too, if it's possible. @tnowacki did you mean to say that system txns allow for arbitrary struct/UID creations?

Reply:
I made the changes and we can making the UID directly with the all powerful system txn.

I can revert all changes I made in object.move, however this means object.move no longer contain all the static ids, is this ok?